sale_order_secondary_unit/tests/test_sale_order_secondary_unit.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2018 Tecnativa - Carlos Dauden
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo.tests import SavepointCase, tagged
+
+
+@tagged('post_install', '-at_install')
```

Comment:
You can remove tags

Reply:
No, it won't work without them, as `sale_stock` is installed before this module and it raises the error `picking_warn` not set when creating products in the test.

Reply:
Ok.ok